task/github-app-token/0.1/github-app-token.yaml

```diff
@@ -65,7 +65,12 @@ spec:
         from jwcrypto import jwk, jwt
 
         EXPIRE_MINUTES_AS_SECONDS = int(os.environ.get('GITHUBAPP_TOKEN_EXPIRATION_MINUTES', 10)) * 60
-        # TODO support github enteprise
+
+        # GitHub Enterprise User
+        # HOST_URL = ""
+        # GITHUB_API_URL = f"{HOST_URL}/api/v3"
+
+        # GitHub User
         GITHUB_API_URL = "https://api.github.com"
```

Comment:
what about making GITHUB_API_URL a params, so user can set this up to GitHub enterprise URL ? like done in the other tasks : 

https://github.com/tektoncd/catalog/blob/a1cd5f81d22efd394c3ba1e12c55e3e8b88e09df/task/github-add-comment/0.2/github-add-comment.yaml#L17-L21